yarn-project/acir-simulator/src/client/private_execution.ts

```diff
@@ -202,8 +203,10 @@ export class PrivateFunctionExecution {
       this.context.request.txContext.contractDeploymentData.portalContractAddress,
 
       this.context.historicRoots.contractTreeRoot,
```

Comment:
The ordering issues might be nice to add a small note around here, would be the first place people run into it.